### ISSUE TITLE
GameruserSettings.ini overwritten

### ISSUE THREAD

**Issue by Murdock2k** [FezVrasta/ark-server-tools#260]
Hey Guys,

everytime i start the Server, he overwrite my Servername/Slot and the passwords...
i changed these commands in the arkmanager.cfg and in the GameuserSettings.ini
but nothing happens, always Ark Server Tools name and 70 slots -.-
I use ubuntu 14.04 and a fresh install...

**Comment by klightspeed** [comment]
It is the ARK server itself that overwrites the `GameUserSettings.ini` file.

If you want to specify these settings (`ServerPassword`, `ServerAdminPassword`, `SessionName`, `MaxPlayers`) in your `GameUserSettings.ini` file, then you need to comment out (i.e. prepend the line with a `#` character) the `ark_ServerPassword`, `ark_ServerAdminPassword`, `ark_SessionName` and `ark_MaxPlayers` settings in the `/etc/arkmanager/arkmanager.cfg` file.

**Comment by Murdock2k** [comment]
Hey,

i have specified it in the GemuserSettings "AND" in the arkmanager.cfg.
After that doesnt work i comment out the variables in the arkmanager.cfg and changed it 
a few times in the GameuserSettings.ini but he always overwrite it...

the Steam user has rights on the gameusersettings.ini and the rootuser on the arkmanager cfg,
so that should be alright, but why ignores the server my commands and change it back to the
defaults?

**Comment by klightspeed** [comment]
Did you perform a user-install (i.e. `curl -sL http://git.io/vtf5N | bash -s -- --me`) or a system-wide install (i.e. `curl -sL http://git.io/vtf5N | sudo bash -s steam`)?

Does `~/.arkmanager.cfg` exist?  If both that file and `/etc/arkmanager/arkmanager.cfg` exist, and you don't want the settings in `~/.arkmanager.cfg` to override those in `/etc/arkmanager/arkmanager.cfg`, then remove `~/.arkmanager.cfg`.  Note that a user-install (i.e. using the `--me` argument to the install) would have created that file.

**Comment by Murdock2k** [comment]
hmm... now the server dont start and i got errors like:
"[  ERROR  ]     You have not rights to write in the log directory.
The server is starting...
/usr/local/bin/arkmanager: line 591: /var/log/arktools/arkserver.log: Permission denied
/usr/local/bin/arkmanager: line 592: /var/log/arktools/arkmanager.log: Permission denied"

i deletet the "~/.arkmanager.cfg" :-(

**Comment by klightspeed** [comment]
What are the permissions on `/var/log/arktools`?

Did you perform a system-wide install with another user, then perform a user-install with this user?

**Comment by Murdock2k** [comment]
hmm... i think i did the install first with the "-s steam" command and than with the "-s -- --me" command -.-
shit... now? Reinstall the mod?

**Comment by klightspeed** [comment]
You should be able to change the owner of `/var/log/arktools` to the user you're running under (i.e. run as root: `chown <username> /var/log/arktools`), and update the `steamcmd_user` setting in `/etc/arkmanager/arkmanager.cfg` to be the username of that user.

**Comment by Lihis** [comment]
Could someone confirm the permission issue too? As I fired up Debian 8.2 in virtualbox to test some mods for my server; I used `curl -sL http://git.io/vtf5N | sudo bash -s steam` to install arkmanager. On fresh arkmanager v1.4 install the `/var/log/arktools` is owned by root and by group root, which obviously causes the permission error.

Meanwhile on production server (Debian 8.2), where I upgraded the arkmanager to v1.4 there is no permission issues as log directory is owned by steam and by group root.
